.github/CONTRIBUTING.md

```diff
@@ -1,58 +1,95 @@
 # Contributions
 
-ClassicPress is a volunteer-driven open source project that thrives when we have multiple contributors working towards a common goal.
+ClassicPress is a volunteer-driven open source project that thrives when we have multiple contributors working towards a common goal. 
+
+We invite all contributors to not only submit pull requests that are in line with the goals and direction of the project, but also to discuss and review other contributors' issues and pull requests. This will help us make the most of our limited time and review all contributions quickly.
 
 Before contributing to this repository, please read our [Democracy page](https://www.classicpress.net/democracy/) to understand how we make decisions about what is included in ClassicPress, and our [roadmap](https://www.classicpress.net/roadmap/) to see what's already planned for the next couple of releases.
 
 The **petitions process** mentioned in our Democracy document is very important to help us prioritize new features based on the needs of our users. There are some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions) to this process for minor changes and bugfixes, but generally speaking it is a good idea to search or ask in one of our communication channels (see below) before undertaking a change, because most changes should go through the petitions process.
 
 Also, please be sure to follow our [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) in all interactions with ClassicPress community members.
 
+## Quick Links
```

Comment:
```suggestion
## Table of Contents

```

"Quick Links" makes it sound like this goes out to another part of the document.

Reply:
Debated it too. Committed. Thanks.